easybuild/easyconfigs/t/torchtext/torchtext-0.14.1_deps_fix.patch

```diff
@@ -0,0 +1,53 @@
+diff -u text/CMakeLists.txt.orig text/CMakeLists.txt
```

Comment:
Please put a comment at the start to explain what the patch does (e.g. the error you get if the patch is not applied). If there is an upstream issue, please include a reference to that as well. That way we can more easily check when the patch might become irrelevant in future versions.